### ISSUE TITLE
WebHook registration

### ISSUE THREAD

**Issue by pbadun** [rubenlagus/TelegramBots#1032]
Good afternoon.
Tell me, does this method register a webhook on the telegram server?
`
@Bean
    public SetWebhook setWebhookInstance() {
        return SetWebhook.builder().url(botConfig.getWebHookPath()).build();
    }
`
Does it make a request like? 
`https://api.telegram.org/bot{api_key}/setWebhook?url=https://site/tg`
Or do you have to do it yourself?

**Comment by loolzaaa** [comment]
Yeah, while you register your bot in `TelegramBotsApi` it invoke `setWebhook` ([link](https://github.com/rubenlagus/TelegramBots/blob/c8ad62b85a1b217f34bd43dd7e30e992227b9a8a/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/TelegramBotsApi.java#L99)) method for Bot. It has default implementation in `WebHook` bot class [(link)](https://github.com/rubenlagus/TelegramBots/blob/c8ad62b85a1b217f34bd43dd7e30e992227b9a8a/telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramWebhookBot.java#L26). After that, `WebhookUtils` create and make a request ([link](https://github.com/rubenlagus/TelegramBots/blob/c8ad62b85a1b217f34bd43dd7e30e992227b9a8a/telegrambots/src/main/java/org/telegram/telegrambots/util/WebhookUtils.java#L49-L103))

**Comment by pbadun** [comment]
I didn’t understand something, but can I have an example of the implementation of behook registration? And at what moment when starting Spring it needs to be applied?

**Comment by varrivoda** [comment]
Sorry I encountered the same issue, so any information I found on the internet is about using the "classic" library, without telegram-springboot-starter, where I call registerBot() manually and and pass it the custom SetWebhook instance I just created in the config file. How do I manage webhook registration if I'm using a spring-boot-starter?
